sdk/identity/identity/src/credentials/deviceCodeCredential.ts

```diff
@@ -3,10 +3,10 @@
 
 import qs from "qs";
 import { TokenCredential, GetTokenOptions, AccessToken } from "@azure/core-http";
+import * as coreHttp from "@azure/core-http";
```

Comment:
Any reason to add this import instead of adding `delay` to the line preceding it where we are importing from `core-http` anyway?

Reply:
If you mean `import { delay } from "@azure/core-http";` I can do that. I'll make a new PR.

Reply:
I mean updating the previous line to `import { TokenCredential, GetTokenOptions, AccessToken, delay  } from "@azure/core-http";`